platform/javascript/detect.py

```diff
@@ -12,7 +12,7 @@ def get_name():
 
 
 def can_build():
-    return 'EMSCRIPTEN_ROOT' in os.environ or 'EMSCRIPTEN' in os.environ
+    return 'EM_CONFIG' in os.environ or os.path.exists(os.path.expanduser('~/.emscripten'))
```

Comment:
Does that work on Windows too?

Reply:
Yeah, Emscripten writes this file to `\Users\%USERNAME%\.emscripten` on Windows, which this call expands to.